yaksh/templates/yaksh/post_comments.html

```diff
@@ -0,0 +1,62 @@
+{% extends "user.html" %}
+
+{% block title %}
+    {{post.title}}
+{% endblock title %}
+
+{% block content %}
+    <div class="container">
+        <a class="btn btn-primary" href="{% url "yaksh:course_forum" post.course.id %}">Back to Posts</a>
+        <div class="card">
+            <div class="card-header">
+                {{post.title}}
+                <br>
+                <small>
+                    <strong>{{post.creator.username}}</strong>
+                    {{post.created_at}}
+                    {% if user.profile.is_moderator %}<a href="{% url "yaksh:hide_post" post.course.id post.uid %}" class="pull-right">Delete</a>{% endif %}
+                </small>
+
+            </div>
+            <div class="card-body">
+                <p>{{post.description}}</p>
+                <a href="{{post.image.url}}" target="_blank">
+                    <img src="{{post.image.url}}"  alt="">
+                </a>
```

Comment:
Set width attribute as 100%. Also, I am getting `ValueError` if the image is not uploaded.